include/zephyr/net/wifi.h

```diff
@@ -536,6 +536,40 @@ static inline const char *get_ps_config_err_code_str(int16_t err_no)
 	return "<unknown>";
 }
 
+/** Promiscuous mode operational settings */
+enum wifi_promiscuous_mode {
+	/** Disable promiscuous Mode setting */
+	WIFI_PROMISC_DISABLE = 0,
+	/** Connection less sniffer mode setting enable */
+	WIFI_PROMISC_CONN_LESS_MODE,
+	/** STA sniffer mode setting enable */
+	WIFI_PROMISC_STA_MODE,
+	/** AP sniffer mode setting enable */
+	WIFI_PROMISC_AP_MODE,
+	/** STA-AP sniffer mode setting enable */
+	WIFI_PROMISC_STA_AP_MODE,
+};
+
+/** Promiscuous mode filter settings */
+enum wifi_promiscuous_filter {
+	/** Support mgmt/data and ctrl packet sniffing */
+	WIFI_PROMISC_ALL = 1,
```

Comment:
```suggestion
	WIFI_PROMISC_ALL = BIT(1),
```
There are more readable.

Reply:
will address the change